packages/contracts/test/test-utils/dao.ts

```diff
@@ -1,8 +1,11 @@
 import {ethers} from 'hardhat';
+import { DAO } from '../../typechain';
```

Comment:
adjust the formatting to the other imports

Reply:
https://github.com/aragon/core/pull/154/commits/f636ab807bfb30abbc1c0f33ec0a2a52190f02f0